modular_citadel/code/modules/clothing/under/polychromic_clothes.dm

```diff
@@ -28,6 +28,7 @@
 	tertiary_color = "#808080"
 	can_adjust = FALSE
 	mutantrace_variation = NO_MUTANTRACE_VARIATION // because I'm too lazy to port these to digi-compatible and to prove a point from /tg/ whining - Pooj
+	force_alternate_icon = TRUE	//in the rare case that someone actually decides to make digi-compatible stuff, this should come in handy
```

Comment:
No! 
Set mutantrace_variation to `MUTANTRACE_VARIATION`, change `worn_overlays()` to use icon_file instead of alternate_worn_icon, and move the digitigrade sprites and overlays to `uniform_digi.dmi` instead! without the damn `_d` suffix!